docs/datafeed.md

```diff
@@ -0,0 +1,121 @@
+# Datafeed
+
+The datafeed service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
```

Comment:
> For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.

Not only, you can also handle connection requests, when a wall post is shared, a form reply